app/controllers/api/mixins/chargeback_assignment.rb

```diff
@@ -104,35 +100,37 @@ def chargeback_rate(parameter_record)
         @chargeback_rate[rate_id] ||= ChargebackRate.find(rate_id)
       end
 
-      def tag_target_assignment(record, _assignment_type, rate_type)
-        target = tag_assigment(record, rate_type)
-        [target, rate_type == "Storage" ? 'storage' : record['assignment_prefix']]
```

Comment:
the format of the return from this method changed.  Were all the callers updated?

Reply:
these only have one caller that is within this file:
tag_target_assignment, label_target_assignment, resource_target_assignment